lib/ood_core/job/adapters/pbspro.rb

```diff
@@ -44,15 +47,22 @@ class Batch
           # @return [Pathname, nil] path to pbs executables
           attr_reader :exec
 
+          # Optional overrides for PBS Pro client executables
+          # @example
+          #  {'qsub' => '/usr/local/bin/qsub'}
+          # @return Hash<String, String>
+          attr_reader :custom_bin
```

Comment:
I wonder if calling this `custom_bin_paths` would be more readable. When I was just glancing at the code, especially the implementation of `OodCore::Job::Adapters::Helper.bin_path` it wasn't clear to me that `bin` and `custom_bin` were two different types dealing with different directories. Though `custom_bin_paths` isn't that much clearer. Any ideas on a better name? if not we can keep it as is.